Lib/test/test_asyncio/test_unix_pipes.py

```diff
@@ -0,0 +1,103 @@
+"""
+Functional tests for Unix transport pipes, designed around opening two sockets
+on the localhost and sending information between
+"""
+
+import unittest
+import sys
+import os
+import tempfile
+import threading
+
+if sys.platform == 'win32':
+    raise unittest.SkipTest('UNIX only')
+
+import asyncio
+from asyncio import unix_events
+
+
+def tearDownModule():
+    asyncio.set_event_loop_policy(None)
+
+
+class UnixReadPipeTransportFuncTests(unittest.TestCase):
+    """
+    Verify that transports on Unix can facilitate reading and have access to
+    all state methods: verify using class internals
+    """
```

Comment:
Style nit: As per PEP 257, within the standard library, we use the following format for all docstrings:

```
"""..."""
```
or
```
"""...
...
"""
```

Rather than: 
```
"""
...
"""
```

I was originally hesitant to point it out for a new test file that doesn't have an existing convention, but during this year's virtual core dev sprint (Oct. 19 - 23), I asked in the general chat and Guido commented that he "unwaveringly in favor" of using the PEP 257 docstring conventions.